src/components/ItemList.styles.js

```diff
@@ -0,0 +1,53 @@
+import { StyleSheet } from 'react-native';
+import colors from '../assets/colors';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    flex: 1,
+    backgroundColor: colors.white,
+    borderBottomWidth: 1,
+    borderBottomColor: '#edeaea',
```

Comment:
probably you just missed this harcoded value, but it would be a good idea adding it to the `Colors` helper